Index: uboot-source/include/configs/stm32mp1.h
===================================================================
--- uboot-source.orig/include/configs/stm32mp1.h
+++ uboot-source/include/configs/stm32mp1.h
@@ -49,7 +49,7 @@
  * Env parameters
  */
 #define CONFIG_ENV_OVERWRITE
-#define CONFIG_ENV_SIZE				SZ_8K
+#define CONFIG_ENV_SIZE				SZ_16K
 
 #if defined(CONFIG_ENV_IS_IN_UBI)
 #define	CONFIG_ENV_UBI_PART			"UBI"
